release-notes/v1.2.6.yaml

```diff
@@ -0,0 +1,23 @@
+date: January 23, 2025
+
+# Changes that are expected to cause an incompatibility with previous versions, such as deletions or modifications to existing APIs.
+breaking changes: |
+
+# Updates addressing vulnerabilities, security flaws, or compliance requirements.
+security updates: |
+  Fixed vulnerability which exposed the Envoy admin interface through the prometheus stats endpoint.
```

Comment:
can we embed the CVE number here ? CVE-2025-24030 , the link doesnt seem to have generated yet, we can link out to https://github.com/envoyproxy/gateway/security/advisories/GHSA-j777-63hf-hx76 for now